src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -108,72 +115,95 @@ public bool TryUnparse(StringBuilder sb)
             }
         }
 
-        public sealed class ColInfoEx
+        public sealed class ColInfo
         {
+            public readonly string Input;
+            public readonly string Output;
             public readonly WhiteningKind Kind;
-            public readonly Float Epsilon;
+            public readonly float Epsilon;
             public readonly int MaxRow;
             public readonly bool SaveInv;
             public readonly int PcaNum;
-            public readonly VectorType Type;
 
-            public ColInfoEx(Column item, Arguments args, ColInfo info)
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="input">Name of the input column.</param>
+            /// <param name="output">Name of the column resulting from the transformation of <paramref name="input"/>. Null means <paramref name="input"/> is replaced. </param>
```

Comment:
>column [](start = 58, length = 6)

Is "one input-output column pair" better?

Reply:
Yes thank you!

---
In reply to: [229524517](https://github.com/dotnet/machinelearning/pull/1452#discussion_r229524517) [](ancestors = 229524517)